components/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+import { ChevronIcon } from './ChevronIcon';
+import styles from '/styles/Button.module.scss';
+
+type ButtonProps = {
+    className?: string;
+    url: string;
+    jobs?: number;
+    children: React.ReactNode;
+};
+
+export const Button = ({ className, url, jobs, children }: ButtonProps) => (
+    <Link href={url}>
+        <a
+            className={
```

Comment:
This might be a nice alternative `[styles.button, className].join(' ')`

I checked with `console.log(["button_class", undefined].join(' '));` and this is from the MDN docs:
_"Warning: If an element is undefined, null or an empty array [], it is converted to an empty string."_


FYI I think you can also pass in attributes as undefined too
i.e. `<div className={undefined}</div>` shouldn't complain

Reply:
This is great, thanks!